_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+license you cannot use it legally. Therefore, getting the license
+right is very important.
+
+**In order to change the MDAnalysis license, we need the agreement of
+all our contributors to change from GPL to LGPL.**
+
+In this post we want to share the motivation, outline the relicensing
+process, and invite questions from the community.
+
+
+
+## Why not GPL any more?
+
+Since its initial release in 2008, MDAnalysis has grown from a small
+Python package used by a handful of enthusiastic graduate students and
+postdocs to a mature library that is used by thousands of researchers
+in the molecular sciences. The MDAnalysis library was published under
+an open source license from the start so that anyone could freely use
+it, contribute to it, and build on it. We chose the GNU Public License
+(GPL) for this purpose. Thanks to the "copy-left" portion of the GPL,
+which requires anyone using MDAnalysis in their own code to also adopt
+the GPL for their code, contributors could feel that any time and work
+that they donated and invested into MDAnalysis would not end up
+contributing to software that they could not use themselves in the
+future.
+
+However, the GPL has also created barriers to adoption of MDAnalysis
+in that it prevented developers to make their own code available under
+other permissive open source licenses. Ultimately, we do not want to
+tell our developers and users how to license their code but we also
+want to clearly signal that the work on MDAnalysis itself is
+guaranteed to remain open and free.
+
+
+## Why now the LGPL?
+
+We therefore decided to start the process to **relicense MDAnalysis
+under the Lesser GNU Public License v3 (LGPL)**. This open source
+license fulfills a number of important requirements for us:
+
+1. Downstream codes are able to freely import or link to MDAnalysis
+   library components without impacting the license choice of the
+   downstream code.
+   
+2. Downstream codes are able to use and subclass any MDAnalysis components
+   under its application programming interface (classes, methods, and
+   data objects), without impacting the license choice of the
+   downstream code.
+   
+3. Codes which either copy or extend the MDAnalysis library should
+   fall under the copyleft license requirements of the MDAnalysis
+   library license.
+   
+Thus, the LGPL gives people the freedom to choose any license for
+their own code that *makes use* of the MDAnalysis library as a whole
+(namely ``import MDAnalysis`` or subclassing). They can even make a
+commercial product out of it with a restrictive license (although we
+encourage open source licenses). However, one cannot just take parts
+of the MDAnalysis code and put it into another piece of code unless
+the other code is then *also* licensed under the GPL/LGPL. The LGPL
+ensures that the MDAnalysis library itself is guaranteed to remain
+open and free.
+
+We considered other licenses but none fulfilled our requirements.
```

Comment:
todo: reword, we don't want to say that the conversation is dead, otherwise why have a community discussion

Reply:
Well.... we are not looking for a discussion about what alternative license we should take. In my view, we did our due diligence and came up with a solution. This is more about building understanding and explaining.

It is more — as you wanted to add above — if the community does not like LGPL then we will remain GPL.

Reply:
If there's any disagreement on this point amonst @MDAnalysis/coredevs then we better get on a call soon.

Reply:
@orbeckst I'm not sure it's a disagreement, with all the other points, it's a case of adjusting language to not shut the door. I'm not saying this specific sentence needs rewording (this is a todo, otherwise I'd just have rewritten it there and then), but it's a "something that needs to be considered when considering the tone of this section" (this is true for all other todos here)